test/browser/features/BrowserBasedTests/UnhappyPath/CMAChangeButton/F2FCheckAnswersIDSelectionChange.feature

```diff
@@ -3,7 +3,7 @@
 
 
 #     Background:
-#         Given Authenticatable Anita is using the system
+#         Given A UK Drivers Licence User is using the system
```

Comment:
I can't see a comment about why this test is commented out - do you know why?

Reply:
It shouldn't have been 😬 uncommented now

Reply:
I don't remember commenting this out, but I uncommented and the test is failing at the first step Given A UK Drivers Licence User is using the system/Given Authenticatable Anita is using the system

Just had a look at main and it's commented out there as well 

https://github.com/govuk-one-login/ipv-cri-f2f-front/blob/main/test/browser/features/BrowserBasedTests/UnhappyPath/CMAChangeButton/F2FCheckAnswersIDSelectionChange.feature

Reply:
Raised https://govukverify.atlassian.net/browse/KIWI-1814 to fix this test